### DISCUSSION TITLE
How do i get the heart on my sleeve badge on github?

### DISCUSSION THREAD

**Discussion by peme969** [community/community#147160] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i think the title is self-explanatory

**Comment by davevad93** [comment]
Hi @peme969 ,

The achievement `Heart On Your Sleeve` isn't yet available to the public, so it isn't yet possible to get it.

**Comment by peme969** [comment, marked as answer]
Oh okay! Thanks :)
(solved)
